### PR TITLE
Repo setup

### PR DESCRIPTION
**Pull request by haukb**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+
+---
+**System information (please complete the following information):**
+ - OS: [e.g. Darwin/18.0.0-x86_64-64bit]
+ - Python Version: [e.g. CPython/3.7.2;Clang 10.0.0]
+ - gql-pygen Version: [e.g. 0.13.0.a4]
```
Comment (doctrino) [inline .github/ISSUE_TEMPLATE/bug-report.md:11]:
```suggestion
 - cognite-powerops Version: [e.g. 0.13.0]
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Description
+Please describe the change you have made.
+
+## Checklist:
+- [ ] Tests added/updated.
+- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
+  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
```
Comment (doctrino) [inline .github/pull_request_template.md:7]:
```suggestion
- [ ] Documentation updated. 
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
```
Comment (doctrino) [inline .github/pull_request_template.md:8]: Update URL

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+- [ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).
```
Comment (doctrino) [inline .github/pull_request_template.md:9]: Update URLs

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+[tool.ruff]
+line-length = 120
+target-version = 'py39'
+
+[tool.poetry.dependencies]
+python = "^3.9"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.3.1"
+pre-commit = "^3.2.2"
```
Comment (doctrino) [inline pyproject.toml:29]: `twine` is missing. Necessary to push to pypi.org

Comment (doctrino) [reply pyproject.toml:29]: `pytest-cov` is also missing

**Review by doctrino — CHANGES_REQUESTED** [review]
A few copy-paste errors.

File: pyproject.toml (hunk 5 above)
Comment (haukb) [reply pyproject.toml:29]: added now

**Review by doctrino — APPROVED** [review]

**Pull request merged**